metrics/web/test/common/presentation/navigation/state/navigation_notifier_test.dart

```diff
@@ -247,6 +273,92 @@ void main() {
       },
     );
 
+    test(
+      ".handlePageParametersUpdates() does nothing if the given page parameters model is null",
+      () {
+        when(pageParametersFactory.create(any)).thenReturn(pageParametersModel);
+
+        notifier.handleAuthenticationUpdates(isLoggedIn: true);
+        notifier.push(DefaultRoutes.dashboard);
+
+        final initialPageParameters = notifier.currentPageParameters;
+
+        notifier.handlePageParametersUpdates(null);
+
+        expect(notifier.currentPageParameters, equals(initialPageParameters));
+      },
+    );
+
+    test(
+      ".handlePageParametersUpdates() does nothing if the given page parameters model is equal to the current page parameters",
```

Comment:
✅ 
I believe we should call the `handlePageParametersUpdates` method somewhere in this test.